builtin/game/chat.lua

```diff
@@ -344,7 +342,7 @@ local function handle_revoke_command(caller, revokename, revokeprivstr)
 		has_irrevokable_priv = true
 	end
 	if privs_unknown ~= "" then
-		return false, privs_unknown
+		core.chat_send_player(caller, privs_unknown)
```

Comment:
shouldn't this still return?

Reply:
Depends on how you want it to function. This way, admins could revoke a set of old privileges without needing to check whether the player has them at all. Of course, such operations would better be done in a mod than manually.